contributor/making-release.md

```diff
@@ -199,15 +205,18 @@ if this is the latest release of Accumulo.
 Some good references that explain a few things:
 
 - [Christopher talks about making releases][3]
```

Comment:
Oh wow, I just clicked on this link and this is me talking about releases when we were using SVN. We haven't done that in a *VERY* long time. This information is at least 7 years old, and almost entirely irrelevant to how we release today.